test-support/helpers/paper-basic-dropdown.js

```diff
@@ -0,0 +1,20 @@
+import wait from 'ember-test-helpers/wait';
+import { click } from 'ember-native-dom-helpers';
+
+export function clickOption(scope, index, options = {}) {
+  return click(`[data-option-index="${index}"]`, scope, options);
+}
+
+export function clickTrigger(scope, options = {}) {
+  if (scope && scope.classList.contains('ember-basic-dropdown-trigger')) {
+    return click(`#${scope.getAttribute('id')}`, options);
+  } else {
+    return click('.ember-basic-dropdown-trigger', scope, options);
+  }
+}
+
+export function selectOption(scope, index) {
```

Comment:
Agree that async/await is easier to read but not sure it was allowed since there is 0 occurence in ember-paper.

I started with a wait() because it was already there in tests (a reason why remove it?) and I didn't add wait() after click() because it returns it already returns it ( https://github.com/cibernox/ember-native-dom-helpers/blob/master/addon-test-support/click.js#L37 )

Maybe this instead ?

```javascript
export async function selectOption(scope, index) {
   await wait();
   await clickTrigger(scope);
   return clickOption();
}
```